developing_packages/workspaces.rst

```diff
@@ -76,88 +184,125 @@ Note that nothing will really be installed in the local cache, all the dependenc
     $ conan search
     There are no packages
 
-Also, each dependency generates one *conanbuildinfo.cmake* file and all of them are installed in the *build* folder. You can inspect them to check
-that the paths they define for their dependencies are user folders, not pointing to the local cache.
+.. note::
 
-As defined in the *conanws.yml*, a root *CMakeLists.txt* is generated for us. We can use it to generate the super-project and build it:
+    The package *conanfile.py* recipes do not contain anything special, they are standard recipes. But the packages *CMakeLists.txt*
+    have defined the following:
 
-.. code-block:: bash
+    .. code-block:: cmake
+
+        conan_basic_setup(NO_OUTPUT_DIRS)
 
-    $ cd build
-    $ cmake .. -G "Visual Studio 14 Win64" # Adapt accordingly to your Conan profile
-    # Now build it. You can also open your IDE and build
-    $ cmake --build . --config Release
-    $ ./A/Release/app.exe
-    Hello World C Release!
-    Hello World B Release!
-    Hello World A Release!
+    This is because the default ``conan_basic_setup()`` does define output directories for artifacts as *bin*, *lib*, etc, which is
```

Comment:
Why not leaving the standard `conan_basic_setup()` call and then make the `layout` file match to it? Explanation, if needed, will be closer to the layout file description, not to the CMake one.

Reply:
Good point, I'll try to improve it later, but I am going to submit my review, so it can be merged soon.